courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/tensors-variables.ipynb

```diff
@@ -19,7 +19,7 @@
     "\n",
```

Comment:
<p>The images are there and were rendering before the lab was moved from my branch to master.  The path to the images need to be changed.   Thanks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Updated the path to the images, all images are now rendering properly in the notebook.